app/scripts/directives/lifecycleHook.js

```diff
@@ -123,7 +123,9 @@ angular.module("openshiftConsole")
 
         $scope.valueFromObjects = [];
         $scope.$watchGroup(['availableSecrets', 'availableConfigMaps'], function() {
-          $scope.valueFromObjects = ($scope.availableSecrets || []).concat($scope.availableConfigMaps);
```

Comment:
@jwforres This could push an `undefined` element into the array if `availableConfigMaps` was not initialized yet, causing #2460